pyproject.toml

```diff
@@ -17,7 +17,7 @@ manylinux-i686-image = "manylinux2014"
 # Change in future version to "build"
 build-frontend = "pip"
 
-[tool.cibuildwheel.override]
+[[tool.cibuildwheel.overrides]]
```

Comment:
Are these square brackets meant to be doubled?

Reply:
Amazingly yes -- https://cibuildwheel.readthedocs.io/en/stable/options/#examples!

I think I got it wrong initially because it was so unexpected.

Reply:
See https://toml.io/en/v1.0.0#array-of-tables. The "O" is "TOML" is apparently not always that obvious. ;)

Reply:
I can understand the necessity for something like that, but the syntax is awful...